src/main/java/org/jboss/xavier/integrations/rbac/RBACRouteBuilder.java

```diff
@@ -58,7 +58,8 @@ public void configure() throws Exception {
                 .setHeader(RBAC_NEXT_LINK, constant(""))
                 .loopDoWhile(exchange -> exchange.getIn().getHeader(RBAC_NEXT_LINK) != null)
                     .setHeader(Exchange.HTTP_METHOD, constant(HttpMethods.GET))
-                    .setHeader(Exchange.CONTENT_TYPE, constant("application/json"))
+                    .setHeader(Exchange.CONTENT_TYPE, constant("application/json")) // Content-type is a way to specify the media type of request being sent from the client to the server (valid on POST and PUT)
```

Comment:
I know this code is not part of this change but i figured out we're already using `ContentType` class in [MainRouteBuilder](https://github.com/project-xavier/xavier-integration/blob/master/src/main/java/org/jboss/xavier/integrations/route/MainRouteBuilder.java#L356).
```suggestion
                    .setHeader(Exchange.CONTENT_TYPE, constant(ContentType.APPLICATION_JSON)) // Content-type is a way to specify the media type of request being sent from the client to the server (valid on POST and PUT)
```